src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 3 && days < 7) {
+    total -= 20;
+  }
+
+  if (days >= 7) {
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    total -= 20;
  } else if (days >= 7) {
```

Use `if - else if` construction